xbmc/video/windows/GUIWindowVideoNav.cpp

```diff
@@ -153,19 +158,24 @@ bool CGUIWindowVideoNav::OnMessage(CGUIMessage& message)
             break;
           }
         }
-        if (i >= m_vecItems->Size() && url.GetOption("showinfo") == "true")
+        if (i >= m_vecItems->Size())
         {
-          // We are here if the item is filtered out in the nav window
-          std::string path = message.GetStringParam(0);
-          CFileItem item(path, URIUtils::HasSlashAtEnd(path));
-          if (item.IsVideoDb())
+          SelectFirstUnwatched();
+
+          if (url.GetOption("showinfo") == "true")
           {
-            *(item.GetVideoInfoTag()) = XFILE::CVideoDatabaseFile::GetVideoTag(CURL(item.GetPath()));
-            if (!item.GetVideoInfoTag()->IsEmpty())
+            // We are here if the item is filtered out in the nav window
+            std::string path = message.GetStringParam(0);
+            CFileItem* item = new CFileItem(path, URIUtils::HasSlashAtEnd(path));
```

Comment:
"I am able to manage the lifecycle of my objects by myself"-koying (Im sorry, I couldn't help it;))

I think I've commented on this exact code before..